### ISSUE TITLE
Scans are not shown properly

### ISSUE THREAD

**Issue by LauraErhard** [locdb/locdb-frend#339]
If I open the first scan in the resolve section (ready/OCR_PROCESSED part), I see the whole image with the message, that OCR processing is required.
![1](https://user-images.githubusercontent.com/32642653/43759397-654b2dce-9a1f-11e8-9a6e-9ff6db8ab0df.png)
If I click the next button, I see the first page, but without the boxes and out of order (see the two examples below, the first page is the third page and the last page is the second)
![2](https://user-images.githubusercontent.com/32642653/43759398-656090ec-9a1f-11e8-82cc-af9786651150.png)
![3](https://user-images.githubusercontent.com/32642653/43759394-64fbea0c-9a1f-11e8-9002-23d7bfda9a3e.png)
If I click on the second scan in the resolve section (ready/OCR_PROCESSED part), I see the first scan and the boxes. But if I click through with the next button, I see strange (probably from different pages) boxes. And the scans are out of order. 
![4](https://user-images.githubusercontent.com/32642653/43759395-6517427a-9a1f-11e8-96f2-10da195627ed.png)
![5](https://user-images.githubusercontent.com/32642653/43759396-6530a530-9a1f-11e8-8a69-4e09b8b6b48c.png)

In summary:
- the first page in the resolve section is always the whole pdf (with every page and without boxes)
- clicking on each page in the resolve section seems to show the right order and the right boxes
- clicking with the next button through the different pages, seems to switch the scans and boxes around
- if I clicked through with the next button, I have to click the browser "back" a few times to get back to the resolve overview (this will get really annoying, when we have pdfs with a lot of pages)

**Comment by lgalke** [comment]
Thanks for the detailed feedback, we will have a look into the issues and see what we can do.

- This comes from our fallback-mechanism to display the PDF if the (ocr-processed) PNG is not available. Due to the filename-splits, this does not hold anymore. So we need to verify our procedure to look up the scans and check whether the orig image is still in the scans list. 

- Regarding 2. and 3., we will make sure that there is no difference in either way of navigating 

- An alternative would be to press on 'Resolve' in the navigation bar. Let me know if this is convenient enough.

**Comment by lgalke** [comment]
Small update: there seems to be some issue with the connection from the agenda. As soon as you press reload, the numbers are properly aligned and we dont have the initial (errornous) pdf.

**Comment by lgalke** [comment]
We're working on it. Maybe we can provide a fix already tomorrow.

**Comment by lgalke** [comment]
There is some progress (the initial pdf is not shown anymore) but for some reason the order of the scans is still unstable. We will delay the deployment until that is fixed aswell!

**Comment by lgalke** [comment]
The pages should be now shown properly. Let us know if there are still any issues with it.

**Comment by LauraErhard** [comment]
I just tried to check this. Only one of my attempts to OCR process a scan worked. There I saw, that the scans are now in the right order, but the boxes stay the same (=the boxes from the scan I clicked on first, stay in the other scans).
![1](https://user-images.githubusercontent.com/32642653/43957819-912df282-9ca9-11e8-94f1-a1787be2ec4c.png)
![2](https://user-images.githubusercontent.com/32642653/43957821-914786de-9ca9-11e8-80c0-11182fe2d6a8.png)

And a negative side effect of not showing the initial pdf is that if the scan doesn't get processed, I can't manually start the OCR process again, because the "scans"-button is not available. Can you suppress the initial pdf only when the scan is successfully processed? Or can you position the little "play"-button, which triggers the new OCR processing, somewhere different?!

**Comment by lgalke** [comment]
We will have a closer Look at this PDF Fallback again. Actually we did want to retain that functionality.

Considering the boxes 'stay the same' thing, I created the other specific issue. Unfortunately, that is not resolvable without adapting OCR and back-end.

**Comment by LauraErhard** [comment]
Thank you!
I just uploaded a scan ("Wie viel ökonomische Bildung braucht politische Bildung?" page 166, PPN 490408818). The scan has only one page and I saw it in the OCR processing part, but after that it just disappeared. Could that be another side effect of not showing the initial pdf? Or am I just blind and don't look at the right place?! If I interpret agendash right the scan should have the id: 5b6d882ae70b9b0a2d8fef5d if that is any help.
